### PR TITLE
Merge-RELEASE. Project 2. #1. Week 2

### PR DESCRIPTION
**Pull request by saffchen**

### CODE REVIEW

**Review by pelmegov — CHANGES_REQUESTED** [review]

File: stock-system/src/main/java/saffchen/entities/SatelliteEntity.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    private Long id;
+
+    @Column(name = "name", unique = true, nullable = false)
+    private String name;
+
+    @Column(name="description")
+    private String description;
+
+    @OneToMany(mappedBy = "satellite")
+    private List<ProductEntity> product;
```
[the remark this answers is not in the document]
Comment (saffchen) [reply stock-system/src/main/java/saffchen/entities/SatelliteEntity.java:32]: Fixed

**Comment by pelmegov** [comment]
@fstk1337 could you please fix issues?

**Comment by fstk1337** [comment]
>

**Comment by fstk1337** [comment]
> @fstk1337 could you please fix issues?

Some fixes done.

**Review by pelmegov — APPROVED** [review]

**Pull request merged**